lib/Models/SubscriberCustomField.php

```diff
@@ -63,16 +63,17 @@ static function createMultiple($values) {
   }
 
   static function updateMultiple($values) {
+    $subscriber_ids = array_map(function($value) {
+      return $value[1];
+    }, $values);
+
+    $params = Helpers::flattenArray($values);
```

Comment:
Why was this extracted to a variable and moved here when it is not used anywhere except its original place?